distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1772,11 +1776,14 @@ type Table
        > Example
```

Comment:
IMO our examples should also contain an example showing that one can have _no_ `group_by` columns and thus get an 'overall' aggregate result with 1 row.